### PR TITLE
Fix KeyError exception when project has no logs

### PR DESCRIPTION
**Pull request by danasmera**
Script was throwing "KeyError: 'logs'" when the project does not have any logs.

### CODE REVIEW

**Comment by danasmera** [comment]
I signed it!

**Comment by waprin** [comment]
Can you provide instructions on how to reproduce this? Even when I create a brand new project I have some logs:

waprin-macbookpro:api waprin$ python list_logs.py core-shard-123708
/projects/core-shard-123708/logs/cloudaudit.googleapis.com%2Factivity

File: cloud_logging/api/list_logs.py [hunk 1]
```diff
@@ -36,6 +37,9 @@ def list_logs(project_id, logging_service):
 
     while request:
         response = request.execute()
+        if len(response)==0:
```
Comment (jonparrott) [inline cloud_logging/api/list_logs.py]: please use ``if not response:``

File: cloud_logging/api/list_logs.py [hunk 2]
```diff
@@ -36,6 +37,9 @@ def list_logs(project_id, logging_service):
...
+            print "No logs found in project {0}".format(project_id)
```
Comment (jonparrott) [inline cloud_logging/api/list_logs.py]: please use print as a function.

File: cloud_logging/api/list_logs.py [hunk 3]
```diff
@@ -36,6 +37,9 @@ def list_logs(project_id, logging_service):
...
+            sys.exit(1)
```
Comment (jonparrott) [inline cloud_logging/api/list_logs.py]: I'd prefer just ``return``.

**Comment by danasmera** [comment]
Here is an output when run against the default project created when I first signed in to Google Cloud -
$ python cloud_logging/api/list_logs.py striking-gadget-120305
Traceback (most recent call last):
  File "cloud_logging/api/list_logs.py", line 65, in <module>
    main(args.project_id)
  File "cloud_logging/api/list_logs.py", line 54, in main
    list_logs(project_id, logging_service)
  File "cloud_logging/api/list_logs.py", line 39, in list_logs
    for log in response['logs']:
KeyError: 'logs'

$gcloud projects describe striking-gadget-120305
createTime: '2016-01-27T05:04:20.755Z'
lifecycleState: ACTIVE
name: My First Project
projectId: striking-gadget-120305
projectNumber: '161387643828'

**Comment by waprin** [comment]
Alright, I cant' reproduce it but I'll give you the benefit of the doubt, thanks for the PR.

**Pull request merged**